CMakeLists.txt

```diff
@@ -15,14 +15,15 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 # Using Cmake version 3.11 for better CUDA & Fortran support
-cmake_minimum_required(VERSION 3.11)
-
-# location for CMake modules (MDI)
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+cmake_minimum_required(VERSION 3.11 FATAL_ERROR)
```

Comment:
Is the version requirement hard? What if I don't care about cuda?

Reply:
I chose that version requirement a while ago with the expectation of having CUDA support re-instated in MOPAC, which hasn't happened yet. Do you think conditional CMake version requirements are a good idea? Without any CUDA consideration, the minimum would probably drop to when CMake switched to the modern "target_sources" style (3.1) or perhaps when some annoying bug was fixed that I will only notice when attempting to use an older version.

Reply:
Sort of: MOPAC should be standard Fortran code, and requiring a new CMake by default - even if you don't need it - is an extra hassle which should be avoided if it can.